schema.sql

```diff
@@ -10,4 +10,28 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals
-ADD species VARCHAR(100);
\ No newline at end of file
+ADD species VARCHAR(100);
+
+CREATE TABLE owners (
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	full_name VARCHAR(150),
+	age INT
+);
+CREATE TABLE species (
+	id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+	name VARCHAR(100)
+);
+
+ALTER TABLE animals
+DROP species,
+ADD species_id int,
+ADD CONSTRAINT fk_animal_spe FOREIGN KEY (species_id) REFERENCES species(id),
+ADD owner_id INT,
+ADD FOREIGN KEY (owner_id) REFERENCES owners(id);
+
```

Comment:
You should modify the animals table:

- Remove column species:heavy_check_mark:
` - Add column species_id which is a foreign key referencing species table`
` -Add column owner_id which is a foreign key referencing the owners table`